src/routes/(marketing)/marketing-layout.css

```
@@ -232,3 +264,369 @@ body:has(.mkt) {
+	font-weight: 650;
+	letter-spacing: 0.02em;
+	line-height: 1.4;
+	border: 1px solid transparent;
+	white-space: nowrap;
+}
+
+.mkt .mkt-pill i {
+	width: 5px;
+	height: 5px;
+	border-radius: 999px;
+	background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)